src/foam/mlang/mlang.js

```diff
@@ -2604,6 +2604,7 @@ if ( ! ( getArg1().f(obj) instanceof String ) ) return false;
 String arg1 = ((String) getArg1().f(obj)).toUpperCase();
 List props = ((foam.core.FObject) obj).getClassInfo().getAxiomsByClass(PropertyInfo.class);
 Iterator i = props.iterator();
+setCheckingNestedFObject_(false);
```

Comment:
I think we wanted 1-level nested fobject check, instead of checking all nested level.

Reply:
@yij793 try moving this to after line 2615 and re-test.

Reply:
@lchanmann add it to line 2615 works